### PR TITLE
Add start page with hero section

### PR DESCRIPTION
**Pull request by Adrian-FL**
Added start page with a simple hero section.
#Verify instructions
1.Open index.html
2.Verify that the hero section exists with header, paragraph and image
3.Verify that <title> of page is "Freaky Fashion".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Document</title>
```
Comment (rikardmansson) [inline index.html:6]: Change <title> to "Freaky Fashion"

Comment (Adrian-FL) [inline index.html:6]: Title should be "Freaky Fashion"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+    <div class ="Hero">
+<h1>Freaky fabulous!</h1>
+        </div>
```
Comment (Adrian-FL) [inline index.html:11]: Hero should have an immage

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+</body>
```
Comment (rikardmansson) [inline index.html:12]: Add paragraph and image

**Pull request closed without merging**